src/iptvsimple/utilities/FileUtils.cpp

```diff
@@ -132,6 +133,44 @@ bool FileUtils::GzipInflate(const std::string& compressedBytes, std::string& unc
   return true;
 }
 
+bool FileUtils::XzDecompress(const std::string& compressedBytes, std::string& uncompressedBytes)
+{
+  if (compressedBytes.size() == 0)
+  {
+    uncompressedBytes = compressedBytes;
+    return true;
+  }
+
+  uncompressedBytes.clear();
+
+  lzma_stream strm = LZMA_STREAM_INIT;
+	lzma_ret ret = lzma_stream_decoder(
```

Comment:
Tab/space issue here and the next few lines?

Reply:
Done